runtime/metricsview/query.go

```diff
@@ -197,6 +197,7 @@ type TimeRange struct {
 	IsoOffset     string    `mapstructure:"iso_offset"`
 	RoundToGrain  TimeGrain `mapstructure:"round_to_grain"`
 	TimeDimension string    `mapstructure:"time_dimension"` // optional time dimension to use for time-based operations, if not specified, the default time dimension in the metrics view is used
+	TimeZone      string    `mapstructure:"time_zone"`      // optional timezone for time-based operations
```

Comment:
Why add a `TimeZone` field that doesn't get consumed anywhere? Time zones impact results, so it's important they are handled correctly.

There's a reason the `TimeZone` is not passed in `TimeRange` – we only support one time zone per query, hence time zones are handled at the query level with the `Query.TimeZone` field.